server/svix-server/src/queue/redis.rs

```diff
@@ -23,6 +23,10 @@
 //! processed now, and the currently processing queue for tasks that have timed out and should be
```

Comment:
This PR needs a better description. Explain the approach you took and why it's good.